conf/timeseries-filodb-server.conf

```diff
@@ -5,6 +5,19 @@ filodb {
     port = 9042
     partition-list-num-groups = 1
   }
+  spread-default = 1
+
+  # Override default spread for application using override block which will have non metric shard keys and spread. There cannot be a shard label named "spread".
+  spread-override = [
```

Comment:
shall we call this spread-assignment instead of "spread-override" ?
Different terms will help distinguish this from spread override passed in command line or query param in verbal conversations.